데이터베이스 h2 -> MySQL (#146)

green4469 opened the pull request:
# 환경 설정
- docker-compose / docker

# 준비 작업
- docker-compose 파일 가서 녹색 화살표 누르면 IntelliJ 가 알아서 도커 컨테이너를 띄워줄 것임

혹시 IntelliJ 가 docker daemon 을 찾지 못하면 (Unix 기반 머신에서는 socket 파일로 통신하기 때문에 docker.sock 을 못찾았다고 뜸) docker group 에 현재 머신의 사용자를 추가해야함. 

```console
sudo usermod -a -G docker ${자신의 사용자 이름}
```

테스트 1개 제외하고 다 통과했는데, 이 테스트는 원래 실패하던 애인듯

green4469 committed:
feat: MySQL 도커 컨테이너 compose 파일

green4469 committed:
feat: MySQL 커넥터 의존성 추가

green4469 committed:
feat: DbUnit 설정 H2 에서 MySQL 로 수정

green4469 committed:
feat: application.properties H2 에서 MySQL 로 수정

green4469 committed:
chore: 불필요한 주석 제거

ddu0422 reviewed (approved):
DB Timezone KST로 변경 후 머지하겠습니다.

green4469 committed:
feat: 타임존 이슈 해결

MySQL 도커의 타임존을 Asia/Seoul 로 변경하고 application.properties 의
연결 타임존을 Asia/Seoul 로 변경했다.

green4469 committed:
refactor: build.gradle compile 에서 implementation 으로 변경
